src/v2/Apps/Order/Components/MinPriceWarning.tsx

```diff
@@ -22,19 +22,15 @@ export const MinPriceWarning: React.FC<MinPriceWarningProps> = ({
       flow: AnalyticsSchema.Flow.MakeOffer,
       order_id: orderID,
     })
-  }, [])
+  }, [orderID, tracking])
```

Comment:
nit: added missing deps from the hook, checked and it's not triggered more than once